src/App/Fossa/IAT/Types.hs

```diff
@@ -0,0 +1,26 @@
+module App.Fossa.IAT.Types (
+  Fingerprint (..),
+  UserDefinedAssertionMeta (..),
+) where
+
+import Data.Aeson
+import Data.Text
+
+-- | Fingerprint uniquely idenfies a file, derived from its content.
+-- Fingerprints are backed by base16 representations of underlying data.
+newtype Fingerprint = Fingerprint Text
```

Comment:
[nit] we often add a record selector to wrapper newtypes, which would replace the need for `toText` below:
```suggestion
newtype Fingerprint = Fingerprint { unFingerprint :: Text }
```

and now we can use `unFingerprint` everywhere we were using `toText` before